src/main/java/no/nav/tag/sykefravarsstatistikk/api/besøksstatistikk/BesøksstatistikkRepository.java

```diff
@@ -0,0 +1,85 @@
+
+    public void lagreBesøkFraStorVirksomhet(
+            Enhet enhet,
+            Sektor ssbSektor,
+            Næringskode5Siffer næring5siffer,
+            Næring næring2siffer,
+            Sammenligning sammenligning,
+            String cookie
+    ) {
+
+        namedParameterJdbcTemplate.update(
+                "insert into besoksstatistikk_virksomhet " +
+                        "(arstall, kvartal, sykefravarsprosent, sykefravarsprosent_antall_personer, naring_2siffer_sykefravarsprosent, ssb_sektor_sykefravarsprosent, orgnr, organisasjon_navn, antall_ansatte, naring_5siffer_kode, naring_5siffer_beskrivelse, naring_2siffer_beskrivelse, institusjonell_sektor_kode, institusjonell_sektor_beskrivelse, ssb_sektor_kode, ssb_sektor_beskrivelse, cookie) " +
+                        "values (:arstall, :kvartal, :sykefravarsprosent, :sykefravarsprosent_antall_personer, :naring_2siffer_sykefravarsprosent, :ssb_sektor_sykefravarsprosent, :orgnr, :organisasjon_navn, :antall_ansatte, :naring_5siffer_kode, :naring_5siffer_beskrivelse, :naring_2siffer_beskrivelse, :institusjonell_sektor_kode, :institusjonell_sektor_beskrivelse, :ssb_sektor_kode, :ssb_sektor_beskrivelse, :cookie)",
+                new MapSqlParameterSource()
+                        .addValue(ÅRSTALL, sammenligning.getÅrstall())
+                        .addValue(KVARTAL, sammenligning.getKvartal())
+                        .addValue(SYKEFRAVÆRSPROSENT, sammenligning.getVirksomhet().getProsent())
+                        .addValue(SYKEFRAVÆRSPROSENT_ANTALL_PERSONER, sammenligning.getVirksomhet().getAntallPersoner())
+                        .addValue(NÆRING_2SIFFER_SYKEFRAVÆRSPROSENT, sammenligning.getNæring().getProsent())
+                        .addValue(SSB_SEKTOR_SYKEFRAVÆRSPROSENT, sammenligning.getSektor().getProsent())
+                        .addValue(ORGNR, enhet.getOrgnr().getVerdi())
+                        .addValue(ORGANISASJON_NAVN, enhet.getNavn())
+                        .addValue(ANTALL_ANSATTE, enhet.getAntallAnsatte())
+                        .addValue(NÆRING_5SIFFER_KODE, næring5siffer.getKode())
+                        .addValue(NÆRING_5SIFFER_BESKRIVELSE, næring5siffer.getBeskrivelse())
+                        .addValue(NÆRING_2SIFFER_BESKRIVELSE, næring2siffer.getNavn())
+                        .addValue(INSTITUSJONELL_SEKTOR_KODE, enhet.getInstitusjonellSektorkode().getKode())
+                        .addValue(INSTITUSJONELL_SEKTOR_BESKRIVELSE, enhet.getInstitusjonellSektorkode().getBeskrivelse())
+                        .addValue(SSB_SEKTOR_KODE, ssbSektor.getKode())
+                        .addValue(SSB_SEKTOR_BESKRIVELSE, ssbSektor.getNavn())
+                        .addValue(COOKIE, cookie)
+        );
+    }
+
+    public void lagreBesøkFraLitenVirksomhet(String cookie) {
+        namedParameterJdbcTemplate.update(
+                "insert into besoksstatistikk_smaa_virksomheter (antall_smaa_virksomheter, cookie) values (1, :cookie)",
```

Comment:
Hva skal `antall_smaa_virksomheter ` brukes til ? Er det tanken at antall økes dersom vi ser at cookie-en er til en ny bedrift?

Reply:
Nei, det er for å telle antall små virksomheter (ant.ansatte < 5) som er inne i appen